libs/pilight/protocols/433.92/selectplus_doorbell.c

```diff
@@ -0,0 +1,183 @@
+/*
+	Copyright (C) 2020 woutput
+
+	This file is part of pilight.
+	This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
At least you got the license right in one go 👍

Reply:
Thanks